SCons/Tool/msvc.xml

```diff
@@ -371,11 +371,17 @@ loaded into the environment.
 
 <para>
 The valid values for &cv-MSVC_VERSION; represent major versions
-of the compiler, except that versions ending in <literal>Exp</literal>
-refer to "Express" or "Express for Desktop" Visual Studio editions,
-which require distict entries because they use a different
-filesystem layout and have some feature limitations compared to
-the full version.
+of the compiler.  Versions ending in <literal>Exp</literal>
```

Comment:
This would be okay as is.  If you want to engage in further wordsmithing, we could clean up the terminology a bit - the string is variously called "valid value", "selector string" and "SCons Key" (in the table heading). I'd further move the verbiage of express to a note below the table, as I think it's a rare case and distracts a bit from the message being conveyed, so perhaps like this (please fact-check my editorialized "but accepted" at the end, since it might be wrong - not sure I fully undertood.

```
<note>
  <para>
  Versions ending in <literal>Exp</literal>
  refer to historical "Express" or "Express for Desktop"
  Visual Studio editions,
  which had feature limitations compared to the full editions.
  It is only necessary to specify the <literal>Exp</literal>
  suffix to select the express version if both express and 
  non-express editions are installed simulaneously.
  The suffix is unnecessary (but accepted) if only the 
  express edition is installed.
  </para>
</note>
```

Reply:
Hmmm, I see you've further commented, which I haven't observed/read yet...

Reply:
I like it!

Took the ball and hopefully advanced it down the field.

The paragraph before the table is now shorter and there is a note with a two item list following the table.  Added the word "IDE" after Visual Studio in the paragraph concerning installation and build tools.  Used the word "product" in the second note instead of version in one place.  Not sure if that is any better (or correct in the context).

Let me know if you care for the changes.